scriptoocloud/pom.xml

```diff
@@ -18,10 +18,19 @@
 
 		<admin.password>$2a$04$oQo44vqcDIFRoYKiAXoNheurzkwX9dcNmowvTX/hsWuBMwijqn44i</admin.password>
 
-		<db.driver>com.mysql.jdbc.Driver</db.driver>
-		<db.database.url>jdbc-url-to-db</db.database.url>
+		<db.driver>com.mysql.cj.jdbc.Driver</db.driver>
+		<db.database.url>db-url</db.database.url>
 		<db.user>db-user</db.user>
 		<db.password>db-password</db.password>
+        
+		<email.user></email.user>
+		<email.password></email.password>
+		<email.ServerHost></email.ServerHost>
+		<email.ServerPort>2525</email.ServerPort>
+		<email.emailFrom></email.emailFrom>
+		<email.instanceUrl></email.instanceUrl>
+		<email.appName></email.appName>
+ 		<email.tls></email.tls>
```

Comment:
let's put default values in `tls` and `appName`; those will probably be the same for most installations.